src/use-css.test.js

```diff
@@ -152,6 +152,24 @@ describe('useCss()', () => {
       ReactDOM.unmountComponentAtNode(container)
     })
   })
+
+  it(`removes the local styleTag references when the component is unmounted`, () => {
+    const postcss = {
+      id: '8',
+      styles: `[star-wars='8'] .kenobi, [star-wars='8'].kenobi{}`,
+      namespace: 'star-wars',
+    }
+
+    act(() => {
+      ReactDOM.render(<ScopedDiv css={postcss} />, container)
+    })
+    expect(container.innerHTML).toEqual('<div star-wars="8"></div>')
+    expect(styleTags[postcss.styles]).toBeDefined()
+    act(() => {
+      ReactDOM.unmountComponentAtNode(container)
+    })
+    expect(styleTags[postcss.styles]).toBeUndefined()
+  })
```

Comment:
This test will fail without [my fix](https://github.com/CanopyTax/kremling/pull/38/files#diff-bb810878b4158c4a2c20b900491bd6f2R27)